account_asset_analytic/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account asset analytic",
+    "summary": "Adds analytic account per asset",
+    "version": "10.0.1.0.0",
+    "category": "Analytic Accounting",
+    "website": "http://www.tecnativa.com",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "account_asset",
```

Comment:
Yes, but throw this route:
account_asset > account_accountant > Account > analytic